snakePipes/shared/rules/FASTQ.snakefile

```diff
@@ -33,10 +33,18 @@ if downsample:
                 seqtk sample -s 100 {input} {params.num_reads} | pigz -p {threads} -9 > {output}
                 """
 else:
-    rule FASTQ:
+    rule FASTQ1:
         input:
-            indir+"/{sample}{read}"+ext
+            indir+"/{sample}"+reads[0]+ext
         output:
-            "FASTQ/{sample}{read}.fastq.gz"
+            "FASTQ/{sample}"+reads[0]+".fastq.gz"
```

Comment:
Would it also work if we here output to  "FASTQ/{sample}{read}.fastq.gz" i.e like for the ext we filter away the read extension in case of single end? and downstream like in star we only use then "FASTQ/{sample}{read}.fastq.gz" as before?!

Reply:
We can hard code it, that won't make any difference.

Reply:
...and the FASQdownsample rule need to be adjusted as well, otherwise they have different files names

Reply:
Some of this probably never actually worked with SE reads, since we almost never generate SE data.

Reply:
yeah maybe, I think downsampling with SE worked at least for RNA if we set reads=["",""] in the config, the other workflows I don't know

Reply:
`reads` was getting reset, so it ended up not mattering what you put in the config file.